test/URLEndpointListenerTest.cc

```diff
@@ -309,27 +309,44 @@ TEST_CASE_METHOD(URLEndpointListenerTest, "Listener with Cert Authentication", "
         0         // port
     };
     listenerConfig.disableTLS = false;
-    listenerConfig.tlsIdentity = createTLSIdentity(true);
-    
+#if !defined(__linux__) && !defined(__ANDROID__)
+    bool anonymous = false;
+    alloc_slice persistentLabel;
```

Comment:
Any persistent identity used in this test (including the anonymous), we will need to delete it. Otherwise, the test machine will be full of new identities. Not sure if you have done this already.

Reply:
I did it in dozen lines after